src/components/TimeseriesChart/interfaces.ts

```diff
@@ -75,8 +79,9 @@ interface TimeseriesChartPropsBase {
    */
   width?: number;
   /**
-   * Mouse move callback
+   * y domain displaying range (max minimum y values) configurations
    */
+  yDomainScaleOptions?: ScaleOptions | { [key: number]: ScaleOptions };
```

Comment:
`yDomainScaleOptions` -> `ySubDomains` and it should be strictly defined type. In this case, I would stick to some kind of mapping, where the key is timeseries ID and the value – an array of two elements – `[min, max]`